packages/transformer/src/IModelExporter.ts

```diff
@@ -751,7 +760,7 @@ export class IModelExporter {
    * You may override this to load arbitrary json state in a transformer state dump, useful for some resumptions
    * @see [[IModelTransformer.loadStateFromFile]]
    */
-  protected loadAdditionalStateJson(_additionalState: any): void {}
+  protected loadAdditionalStateJson(_additionalState: any): void { }
```

Comment:
is this the linter?

Reply:
I formatted this file from vs code, so it may be the linter